.hadolint.yml

```
@@ -0,0 +1,5 @@
+ignored:
+ - DL3009
```

Can you add comments explaining what these ignored errors are?

---

Handled all of the errors so I removed this file.